src/markdown-pages/build-apps/build-visualization.mdx

```diff
@@ -203,7 +203,7 @@ When your visualization is ready to be added to a dashboard, just follow these s
 
   <Step>
 
-Open and follow the guide to [publish and deploy the Nerdpack to New Relic One and subscribe accounts to it](/build-apps/publish-deploy).
+Open and follow the guide to [publish and deploy the Nerdpack to New Relic One and subscribe accounts to it](/build-apps/publish-deploy). Note that you must first subscribe to your new app, log out and log back in to New Relic One, before being able to view the visualization in the Custom Visualizations app itself (where you saw it running locally).
```

Comment:
Can we change this to `Note that you must first subscribe to your new app before being able to view the visualization in the Custom Visualizations app itself (where you saw it running locally)`